### PR TITLE
[pyyaml][Draft] initial integration.

### PR DESCRIPTION
**Pull request by DavidKorczynski**

### CODE REVIEW

File: projects/pyyaml/build.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd pyyaml
+python3 ./setup.py --without-libyaml install
+#pip3 install .
```
Comment (inferno-chromium) [inline projects/pyyaml/build.sh:19]: remove this.

File: projects/pyyaml/project.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/yaml/pyyaml"
+language: python
+primary_contact: "david@adalogics.com"
```
Comment (inferno-chromium) [inline projects/pyyaml/project.yaml:3]: Lets get a dev contact here.

**Review by inferno-chromium — APPROVED** [review]

**Comment by inferno-chromium** [comment]
Please also fix shebang line, see https://github.com/google/oss-fuzz/pull/4746

File: projects/pyyaml/project.yaml (hunk 2 above)
Comment (ingy) [inline projects/pyyaml/project.yaml:3]: Double quotes are not needed here.

File: projects/pyyaml/project.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+fuzzing_engines:
+  - libfuzzer
+sanitizers:
+  - address
```
Comment (ingy) [inline projects/pyyaml/project.yaml:7]: Better as:
```
fuzzing_engines:
- libfuzzer
sanitizers:
- address
```

File: projects/pyyaml/project.yaml (hunk 2 above)
Comment (ingydotnet) [reply projects/pyyaml/project.yaml:3]: @ingy is me (@ingydotnet)

Comment (DavidKorczynski) [reply projects/pyyaml/project.yaml]: inserted pyyaml@yaml.com supplied from here https://github.com/yaml/pyyaml/issues/464#issuecomment-750445481

File: projects/pyyaml/project.yaml (hunk 3 above)
Comment (DavidKorczynski) [reply projects/pyyaml/project.yaml:9]: I went with the way most projects in oss-fuzz are set up and also how the documentation specifies it https://google.github.io/oss-fuzz/getting-started/new-project-guide/python-lang/#projectyaml

I left it as originally for now due to above reasons - let me know if you would still like the change.

**Comment by DavidKorczynski** [comment]
@inferno-chromium @jonathanmetzman @oliverchang  this one is good to go

**Review by jonathanmetzman — APPROVED** [review]

**Comment by jonathanmetzman** [comment]
LGTM

**Pull request merged**